app/models/item.rb

```diff
@@ -25,4 +25,23 @@ def no_orders?
     item_orders.empty?
   end
 
+  def self.top_5
+    top_5_data = ItemOrder.find_by_sql ["SELECT items.name, item_id, SUM(quantity) AS total_purchased FROM item_orders JOIN items ON item_orders.item_id = items.id GROUP BY item_id, items.name ORDER BY total_purchased DESC LIMIT 5"]
+    # Item.joins(:item_orders)
+    #     .select('items.id, items.name, sum(item_orders.quantity)')
+    #     .group('items.id')
+    #     .order('sum(item_orders.quantity)')
+    #     .limit(5)
```

Comment:
Super Sage and I were chatting about how to do this in ActiveRecord but neither of us were able to get there 100%. Left this wonderful code from Sage in to get hopefully get some feedback on during check-in tomorrow.